model_zoo/census_dnn_model/census_feature_columns.py

```diff
@@ -0,0 +1,56 @@
+import tensorflow as tf
+
+from elasticdl.python.elasticdl.feature_column import feature_column
```

Comment:
The feature_column.py only contains `embedding_column`. Maybe it better to rename feature_column.py to embedding_column.py?